### PR TITLE
Create q_full_adder.py

### PR DESCRIPTION
**Pull request by KevinJoven11**
This is for the #Hacktoberfest.
This circuit is the quantum full adder. I saw that in the repo is the half adder so I decided to build the full adder to complete the set of adders. I hope that this is enough to be consider a contribution. Best,
Kevin

### Describe your change:

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by KevinJoven11** [comment]
Dear @cclauss, 
This is another contribution to the quantum folder. I created the quantum full adder (currently there is half adder in the repository) I only want to complete the set. This is my third contribution. Please let me know if you can review this. After I finish this I will update the "superdense conding" (#6783 ). 
Thanks for your time.

File: quantum/q_full_adder.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        input_2: input 2 for the circuit.
+        carry_in: carry in for the circuit.
+    Returns:
+        qiskit.result.counts.Counts: sum result counts.
+    >>> quantum_full_adder(1,1,1)
+    {'11': 1000}
+    >>> quantum_full_adder(0,0,1)
+    {'01': 1000}
+    >>> quantum_full_adder(1,0,1)
+    {'10': 1000}
```
Comment (cclauss) [inline quantum/q_full_adder.py:44]: Please add some tests that fail...  Negative numbers, floating point numbers, letters, etc.

**Review by cclauss — CHANGES_REQUESTED** [review]

File: quantum/q_full_adder.py (hunk 1 above)
Comment (KevinJoven11) [reply quantum/q_full_adder.py:46]: Done!

File: quantum/q_full_adder.py [hunk 2]
```diff
@@ -9,26 +9,26 @@
 """
 
 import qiskit
-from qiskit import Aer, ClassicalRegister, QuantumCircuit, QuantumRegister, execute
-
+from qiskit import QuantumCircuit, QuantumRegister, ClassicalRegister, execute, Aer
+import math
```
Comment (cclauss) [inline quantum/q_full_adder.py:11]: Kinda cool how pre-commit fixes all this stuff… trailing whitespace, unsorted imports, single quotes, etc.

**Review by cclauss — APPROVED** [review]

**Pull request merged**